pkgs/desktops/xfce/panel-plugins/xfce4-timer-plugin.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, pkgconfig, intltool, libxfce4util, xfce4panel, libxfce4ui, libxfcegui4, xfconf, gtk}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  p_name  = "xfce4-timer-plugin";
```

Comment:
You have to reference the plugin somewhere. Otherwise it will not be picked up by travis.